docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM quay.io/sameersbn/ubuntu:latest
+MAINTAINER Alexander Trost <galexrt@googlemail.com>
+
+ENV ZULIP_VERSION="1.3.7" ZULIP_CHECKSUM="88bfa668eb14e07b0b806977db2ae2cd4d7e7ef8" DATA_DIR="/data"
```

Comment:
@lfaraone The hash is from here https://zulip.org/server.html#download.
Be aware of that I'm **not** cloning the repo into the image nor using a tarball from github, I use the "download" version of zulip from https://zulip.org/server.html#download, I use the hash to verify the file.
But it's not a problem to remove the hash.
Shall I remove the hash from the Dockerfile then?